fe/fe-core/src/main/java/org/apache/doris/catalog/MTMV.java

```diff
@@ -473,6 +473,12 @@ public void setRefreshSnapshot(MTMVRefreshSnapshot refreshSnapshot) {
         this.refreshSnapshot = refreshSnapshot;
     }
 
+    public boolean canBeCandidate() {
+        // MTMVRefreshState.FAIL also can be candidate, because may have some sync partitions
+        return getStatus().getState() == MTMVState.NORMAL
+                && getStatus().getRefreshState() != MTMVRefreshState.INIT;
```

Comment:
How about create a `canBeCandidate()` in `MTMVStatus`?
So that we won't miss this judgement if we add or delete the status or state enum.
For example, if i add a new `MTMVRefreshState`, this logic may be forgot to modify